packages/api/src/controllers/admin/import/ImportWeaponsController.ts

```diff
@@ -43,6 +44,7 @@ export class ImportWeaponsController {
   }
 
   @Post("/")
+  @Description("Import Vehciles in the CAD via file upload")
```

Comment:
```suggestion
  @Description("Import weapons in the CAD via file upload")
```

should be weapons not vehicles here